### DISCUSSION TITLE
Load README.md from URL

### DISCUSSION THREAD

**Discussion by leon-thomm** [docsifyjs/docsify#1644] [Q&A] [answered]
Hey, I would like to embed a Docsify generated page in a simple website I am building using [FastAPI](https://fastapi.tiangolo.com/) (in Flask-like manner). However, when generating the HTML, all URLs for resources need to be resolved (FastAPI will insert the correct URLs for all Jinja2 `url_for()` commands). Therefore, I need to specify the url for the markdown file in the HTML already and cannot wait for Docsify to be loaded in the client's browser. Is there a way to do something like that:

```javascript
            window.$docsify = {
                name: '',
                repo: '',
                src: "{{ url_for('guide', path='/README.md') }}",
            }
```
and not let Docsify search for the file?

Sorry in case the question doesn't make much sense, I'm just getting started with most of this web stuff.
Thanks!

**Comment by sy-records** [comment, marked as answer]
see https://docsify.js.org/#/configuration?id=basepath

**Reply by leon-thomm** [reply]
Ah nice, thank you!
